### PR TITLE
[Backport to 2.0] Fix 'openserach' typo in constants.tsx (#953)

### PR DESCRIPTION
**Pull request by ylwu-amzn**
Signed-off-by: Cam McKenzie <camAtGitHub@users.noreply.github.com>

### Description
Backport PR #953 to 2.0

### Category
bug fix

### Issues Resolved
[List any issues this PR will resolve (Is this a backport? If so, please add backport PR # and/or commits #)]

### Testing
[Please provide details of testing done: unit testing, integration testing and manual testing]

### Check List
- [ ] New functionality includes testing
- [ ] New functionality has been documented
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by cliu123** [comment]
@ylwu-amzn This PR needs to be included in the [release notes](https://github.com/opensearch-project/security-dashboards-plugin/blob/main/release-notes/opensearch-security-dashboards-plugin.release-notes-2.0.0.0-rc1.md) if this will be merged to 2.0. The release notes change also needs to be made in main, 2.x and 2.0 branches.

**Review by peternied — CHANGES_REQUESTED** [review]
This change is not approved for the 2.0.0-rc1 release.

**Comment by ylwu-amzn** [comment]
> 

Thanks @peternied for adding this. We should merge this PR after `2.0-rc1` release.

**Review by davidlago — APPROVED** [review]

**Review by DarshitChanpura — APPROVED** [review]

**Review by peternied — APPROVED** [review]

**Comment by ylwu-amzn** [comment]
I have no permission to merge PR. Security team please help merge.

**Pull request merged**
